Android/firebase_dependencies.gradle

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 // A map of library to the dependencies that need to be added for it.
+import org.gradle.util.*;
```

Comment:
consider change to
`import org.gradle.util.ConfigureUtil;`

Reply:
updated 😉